contrib/test/ci/main.yml

```diff
@@ -1,195 +0,0 @@
----
```

Comment:
What's the reason that we removed that file?

Reply:
This file has not been used for running any of our new CI jobs.